test/pan-and-zoom.spec.ts

```diff
@@ -123,12 +122,12 @@ describe("PanAndZoom", () => {
 					maxPanX: +5,
 					maxPanY: +5,
 				});
-				paz.panX    = [];        expect(paz.panX).to.be.a("number").that.equals(0);
-				paz.panY    = {};        expect(paz.panY).to.be.a("number").that.equals(0);
-				paz.minPanX = undefined; expect(paz.minPanX).to.be.a("number").that.equals(0);
-				paz.minPanY = null;      expect(paz.minPanY).to.be.a("number").that.equals(0);
-				paz.maxPanX = "Nope";    expect(paz.maxPanX).to.be.a("number").that.equals(0);
-				paz.maxPanY = NaN;       expect(paz.maxPanY).to.be.a("number").that.equals(0);
+				paz.panX    = [] as any;        expect(paz.panX).to.be.a("number").that.equals(0);
+				paz.panY    = {} as any;        expect(paz.panY).to.be.a("number").that.equals(0);
+				paz.minPanX = undefined as any; expect(paz.minPanX).to.be.a("number").that.equals(0);
+				paz.minPanY = null as any;      expect(paz.minPanY).to.be.a("number").that.equals(0);
+				paz.maxPanX = "Nope";    		expect(paz.maxPanX).to.be.a("number").that.equals(0);
+				paz.maxPanY = NaN;       		expect(paz.maxPanY).to.be.a("number").that.equals(0);
```

Comment:
Tabs should only be [used for indentation](https://github.com/Alhadis/Coding-Style#1-use-tabs-for-indentation), not for alignment. 😉 That is:

1. Tabs at the start of a line
2. Spaces after non-whitespace characters
3. Never let tabs and spaces touch

Reply:
```suggestion
				paz.maxPanX = "Nope";           expect(paz.maxPanX).to.be.a("number").that.equals(0);
				paz.maxPanY = NaN;              expect(paz.maxPanY).to.be.a("number").that.equals(0);
```